patches/third_party-blink-renderer-modules-payments-payment_request.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/third_party/blink/renderer/modules/payments/payment_request.cc b/third_party/blink/renderer/modules/payments/payment_request.cc
+index 1155e412231e9aeb6dddb1733c8a674182f4fce2..53d18c59fcd37dc4b34888a186199dfba6631af9 100644
+--- a/third_party/blink/renderer/modules/payments/payment_request.cc
++++ b/third_party/blink/renderer/modules/payments/payment_request.cc
+@@ -124,6 +124,7 @@ struct TypeConverter<PaymentItemPtr, blink::PaymentItem> {
+     output->label = input.label();
+     output->amount = PaymentCurrencyAmount::From(*input.amount());
+     output->pending = input.pending();
++    BRAVE_TYPE_CONVERTER_PAYMENT_ITEM
```

Comment:
shouldn't we also validate/sanitize it somehow? I see a bunch of validation code in this file. What if a site posts a megabyte of text into `sku`? or maybe we have a specific format in mind